src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/utils/Constants.java

```diff
@@ -8,5 +8,5 @@ private Constants() { }
     public static final String ERIC_REQUEST_ID_KEY = "X-Request-Id";
 
     // URL path parameters
-    public static final String URL_PARAM_TRANSACTION_ID = "transaction_id";
+    public static final String URL_PARAM_TRANSACTION_ID = "transactionId";
```

Comment:
does the spec need updating for the url param name change

Reply:
Yes it does. Updated the spec.